CHANGELOG.md

```diff
@@ -50,6 +50,8 @@ Main (unreleased)
     drops data or forces a garbage collection if the defined limits are
     exceeded. (@tpaschalis)
 
+- Flow: Allow config blocks to reference component exports. (@tpaschalis)
```

Comment:
Can you update the docs to remove references to the limitation that config blocks can't reference component exports?

Reply:
Yep, done!